src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -729,6 +747,32 @@ static void find_and_replace_parameters(
   }
 }
 
+/// Convert parsed annotations into the symbol table
+/// \param parsed_annotations: The parsed annotations to convert
+/// \param annotations: The java_annotationt collection to populate
+void convert_annotations(
+  const java_bytecode_parse_treet::annotationst &parsed_annotations,
+  std::vector<java_annotationt> &annotations)
+{
+  for(const java_bytecode_convert_classt::annotationt &annotation
```

Comment:
I'd use `auto` as the type name is exceptionally verbose (unless the interface is different).